libraries/common/seq.effekt

```diff
@@ -1,41 +1,32 @@
 module seq
-
-/**
- * This file implements the [[Seq]] type as a general purpose functional sequence.
- *
- * Implemented as a 2-3 finger tree, it supports
- * - prepend and append in amortized O(1),
- * - concat(m, n) in O(log(min(m, n)))
- * - first and last in amortized O(1)
- * - size in amortized O(1)
- *
- * More information on finger trees:
- *   https://www.staff.city.ac.uk/~ross/papers/FingerTree.pdf
- */
-
-/**
- * Sequences of elements
- *
- * They are represented as 2-3 finger trees.
- */
+/// This file implements the [[Seq]] type as a general purpose functional sequence.
+///
+/// Implemented as a 2-3 finger tree, it supports
+/// - prepend and append in amortized O(1),
+/// - concat(m, n) in O(log(min(m, n)))
+/// - first and last in amortized O(1)
+/// - size in amortized O(1)
+///
+/// More information on finger trees:
+///   https://www.staff.city.ac.uk/~ross/papers/FingerTree.pdf
```

Comment:
@BinderDavid have you thought of supporting a module-level doc comment?
Do you think it should be under the `module ...` definition or above it? 🤔

Reply:
ACK, makes sense, thanks